drivers/include/ina3221.h

```diff
@@ -0,0 +1,1094 @@
+/*
+ * Copyright (C) 2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_ina3221 INA3221 current/power monitor
+ * @ingroup     drivers_sensors
+ * @brief       Device driver for Texas Instruments INA3221
+ *              three-channel ,high-side current and bus voltage
+ *              monitor
+ * @{
+ *
+ * @file
+ * @brief       Device driver interface for Texas Instruments INA3221
+ *              three-channel ,high-side current and bus voltage
+ *              monitor.
+ *
+ *              The alert pins WRN, CRT, TC, PV are open drain
+ *              output pins, so they should be pulled high with
+ *              an external pull-up resistor. They pull low, if
```

Comment:
See comment above regarding the MCU-internal pull ups, that could be used instead.